tests/0103-transactions.c

```diff
@@ -1003,6 +1003,10 @@ static void do_test_empty_txn(rd_bool_t send_offsets, rd_bool_t do_commit) {
         else
                 TEST_CALL_ERROR__(rd_kafka_abort_transaction(p, -1));
 
+        /* Wait before checking the committed offsets (Kafka < 2.5.0) */
+        if (test_broker_version < TEST_BRKVER(2, 5, 0, 0))
+                rd_usleep(tmout_multip(2000 * 1000), NULL);
```

Comment:
I think 2s is too short in the standard case too, which will make the test flaky. Let's make it 5s by default.